forwards_paper.tex

```diff
@@ -187,35 +190,40 @@ \section*{Introduction}
 \section*{Results}
 
 
-We begin by benchmarking the performance improvement achieved by connecting
-a previously published forwards-time simulation library,
-\fwdpp{} \citep{fwdpp},
-to the tools we describe here, as implemented in \msprime{}.
+The strategy described above is only of interest if it is computationally feasible.
```

Comment:
trying to say *why* we put the simulation results before saying how they actually work